jest.config.js

```diff
@@ -1,20 +1,11 @@
 module.exports = {
   testEnvironment: 'node',
   transform: {
-    "^.+\\.tsx?$": "ts-jest"
+    '^.+\\.tsx?$': 'ts-jest',
   },
-  moduleFileExtensions: [
-    "ts",
-    "tsx",
-    "js",
-    "jsx",
-    "json",
-    "node",
-  ],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
```

Comment:
Here are well, we can drop tsx and jsx
```suggestion
  moduleFileExtensions: ['ts', 'js', 'json', 'node'],
```

Reply:
👍